### DISCUSSION TITLE
page.add and page.views.append diffrent alignment

### DISCUSSION THREAD

**Discussion by keyiflerolsun** [flet-dev/flet#597] [Q&A]
## My page settings is "center";
![image](https://user-images.githubusercontent.com/57468649/202846405-500b1daf-0e58-4945-8ecd-d146c59b28ac.png)

## `page.add` method is center;
![image](https://user-images.githubusercontent.com/57468649/202846351-a973d2d3-751e-4755-9c67-b5eceac24850.png)

## `page.views.append` method is top;
![image](https://user-images.githubusercontent.com/57468649/202846487-43d04a42-3b27-49e3-8f6d-944c90bbc266.png)

**Comment by mnesarco** [comment]
AFAIK `page.add` is an alias of `page.controls.append` not `page.views.append`
